PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,9 @@ Please fill out the relevant template with as much detail as you can provide.
 
```

Comment:
I think a New Model Proposal is better done as an issue?

We can create an issue template for that.

The code change or feature request is ok for a PR template perhaps, but doesn't it make more sense to have that in each model repo?

Reply:
I agree. See #8